libraries/chain/controller.cpp

```diff
@@ -1592,8 +1607,10 @@ struct controller_impl {
                trace->receipt = r;
             }
 
-            fc::move_append( std::get<building_block>(pending->_block_stage)._action_receipt_digests,
-                             std::move(trx_context.executed_action_receipt_digests) );
+            if ( !trx->is_transient() ) {
+               fc::move_append( std::get<building_block>(pending->_block_stage)._action_receipt_digests,
+                                std::move(trx_context.executed_action_receipt_digests) );
+            }
 
             // call the accept signal but only once for this transaction
```

Comment:
Move this comment down above the `emit`.

Reply:
Done.

Reply:
I think that comment goes with ` if (!trx->accepted) {`.